portfolio/src/main/webapp/script.js

```diff
@@ -81,12 +81,47 @@ function createCommentElement(comment) {
   commentElement.classname = 'comment';
 
   const headerElement = document.createElement('span');
-  headerElement.innerHTML = "<h3>"+comment.posterName +"<h3>";
+  headerElement.innerHTML = "<h3>" + cleanseString(comment.posterName) +"<h3>";
 
   const bodyElement = document.createElement('span');
-  bodyElement.innerHTML = "<p>" + comment.comment +"<p>";
+  bodyElement.innerHTML = "<p>" + cleanseString(comment.comment) +"<p>";
 
   commentElement.appendChild(headerElement);
   commentElement.appendChild(bodyElement);
   return commentElement;
-}
\ No newline at end of file
+}
+
+function cleanseString(html) {
+  // Prevent html injection by replacing escape characters with plaintext
+  return html.replace(/</g, "&lt;").replace(/>/g, "&gt;");
+}
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+function drawChart() {
+  const data = new google.visualization.DataTable();
+  console.error("Hello world!");
```

Comment:
Is this needed?

Reply:
Oops, no it is not, I had put this in for testing purposes but forgot to remove it.